package.json

```diff
@@ -4,13 +4,21 @@
   "description": "Glasgow Graduation Project - Women Shelter system",
   "main": "index.js",
   "scripts": {
-    "start": "npm --prefix server start run dev & npm --prefix client start",
-    "install": "npm --prefix server install & npm --prefix client install"
+    "client": "npm --prefix client start",
+    "server": "npm --prefix server start",
+    "dev": "NODE_ENV=development npm run client & npm --prefix server run dev",
+    "start": "npm run server",
+    "build": "cd client && npm install && npm install --only=dev --no-shrinkwrap && npm run build && cd ../",
```

Comment:
how come we need to "npm install" dev dependencies as well in the build command?

Reply:
That I'm not sure. @kabaros wrote this originally I think.